facilitator/assets/exams/cka/003/scripts/validation/q6_s2_validate_pod_distribution.sh

```
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+# Check if DaemonSet pods are running on all worker nodes
+WORKER_NODE_COUNT=$(kubectl get nodes --no-headers | grep -v "control-plane\|master" | wc -l)
```

Shellcheck (0.10.0)

[warning] WORKER_NODE_COUNT appears unused. Verify use (or export if used externally).

(SC2034)